src/pages/store/index.vue

```diff
@@ -222,6 +318,62 @@
           </li>
         </ul>
 
+        <!-- Search loading -->
+        <div v-else-if="isSearchLoading" class="flex justify-center">
+          <ProgressIndicator />
+        </div>
+
+        <!-- Search result -->
+        <ul
+          v-else
+          :class="[
+            'w-full',
+            'grid',
+            'grid-cols-2',
+            'sm:grid-cols-3',
+            'laptop:grid-cols-3',
+            'desktop:grid-cols-4',
+            'desktopLg:grid-cols-5',
+            'full-hd:grid-cols-6',
+            'gap-x-[16px] sm:gap-x-[20px] gap-y-[40px]',
+            'items-stretch',
+            'desktop:mt-0',
+          ]"
+        >
+          <li v-for="item in sortedBookstoreItems" :key="item.classId">
+            <NFTBookItemCardV2
+              :item-id="item.classId"
+              class-cover-frame-aspect-ratio="aspect-[4/5]"
+              :is-link-disabled="item.isMultiple"
+              @click-cover="handleClickItem($event, item)"
+            />
+          </li>
+        </ul>
+        <!-- Search not found -->
+        <div
+          v-if="searchQuery && !sortedBookstoreItems.length"
```

Comment:
What is `isSearchLoading`?

Reply:
@AuroraHuang22 Actually i wanted to ask if `isSearching` is set, should we hide `Search not found` message?
autocomplete issue...

Reply:
[🚸 show empty state only when search is complete](https://github.com/likecoin/liker-land/pull/1906/commits/47cfed3df4c5305071a7ec68b8d23da280702eb0)